ceno_zkvm/src/scheme/mock_prover.rs

```diff
@@ -240,36 +277,49 @@ impl<E: ExtensionField> MockProverError<E> {
                 } else {
                     "Lookup".to_string()
                 };
-                let location = if *count > 0 {
-                    "constraint system"
+
+                let (location, element) = if let Some(key) = key.to_u64() {
+                    let location = if *count > 0 {
```

Comment:
`count` is recorded to print errors in the following fashion:

```sh
# when you have Uint::new in construct while Value::new_unchecked in assign
LkMultiplicityError:
2 Lookups of U16 missing in assignments
Element: 0

# when you have Uint::new_unchecked in construct while Value::new in assign
LkMultiplicityError:
2 Lookups of U16 missing in constraint system
Element: 0
```

But I agree this could be more self explaining. Instead of count being a signed int, it could be an enum, something like-

```rs
enum LkMultiplicityErrorCount {
    MissingInAssignments(u64),
    MissingInConstraintSystem(u64),
}
```

Reply:
Yes.  Perhaps something for the next PR?